test/functional/feature_fee_estimation.py

```diff
@@ -73,36 +73,38 @@ def small_txpuzzle_randfee(
 
 def check_raw_estimates(node, fees_seen):
     """Call estimaterawfee and verify that the estimates meet certain invariants."""
-
-    delta = 1.0e-6  # account for rounding error
+    delta = Decimal('0.000001')  # account for rounding error
```

Comment:
Agreed might be an overkill because `check_raw_estimates` function tests for a range, ensuring the returned fee rate is sane.

Sounds good to me to limit this maflko suggestion

Reply:
Slight +1 on not making `delta` a Decimal, which suggests there's something important about its precision?

Reply:
I was aiming for exact precision because of this comment https://github.com/bitcoin/bitcoin/pull/23225#issuecomment-938801112, but I get your point. Maybe if fee estimation changes to decimals internally in future, the tests can be modified accordingly.

Reply:
@maflcko  IIUC  reverting to the earlier version would mean that I also need to discard  all the  changes made to the functions `check_raw_estimates` and `check_smart_estimates`, right?